internal/server/openapi.go

```diff
@@ -398,6 +398,20 @@ func buildRequest(r reflect.Type, op *openapi3.Operation, method string) {
 		},
 	})
 
+	op.AddParameter(&openapi3.Parameter{
+		Name:     "Authorization",
+		In:       "header",
+		Required: true,
```

Comment:
It's only required for some paths. We don't have the `route` all the way through to this function yet, and I think doing that is a larger change to this code. Should we set this to `false` until we can correctly set `true/false` on each path?

Reply:
bummer

Reply:
So is it just Login and Signup?

I tried 

```
var noLoginOperations = []string{"Login", "Signup"}
if !slices.Contains(noLoginOperations, op.OperationID) {
  op.AddParameter...
```

Would that solve the problem?

Reply:
Found the list: Login, RequestPasswordReset, VerifiedPasswordReset, GetProvider, ListProviders, GetSettings, Signup, Version, GetServerConfiguration, RequestForgotDomains